client/components/domains/domain-suggestion-flag/index.jsx

```diff
@@ -10,28 +10,24 @@ import { localize } from 'i18n-calypso';
  */
 import Notice from 'components/notice';
 
-class DomainSuggestionFlag extends React.Component {
-	render() {
-		const newTLDs = [ 'blog' ];
+function DomainSuggestionFlag( { domain, translate } ) {
+	const newTLDs = [ 'blog' ];
 
-		if ( newTLDs.some( ( tld ) => {
-			return endsWith( this.props.domain, tld );
-		} ) ) {
-			return (
-				<Notice
-					isCompact
-					status="is-success">
-					{ this.props.translate( 'New', { context: 'Domain suggestion flag' } ) }
-				</Notice>
-			);
-		}
+	if ( newTLDs.some( ( tld ) => {
+		const isSubdomain = domain.substring( 0, domain.length - ( tld.length + 1 ) ).indexOf( '.' ) > -1;
```

Comment:
If we ever add something that's two or three letter longs to the newTLDs array, this will always return `true`, no?

Let's do `domain.split('.').length > 2`, no?